include/net/nrf_cloud.rst

```diff
@@ -22,64 +25,44 @@ Before using any other APIs of the module, the application must call :cpp:func:`
 
 Connecting
 **********
-The application can use :cpp:func:`nrf_cloud_connect` to connect to the cloud. This API triggers a series of events and actions in the system. If the API fails, the application must retry to connect.
-:cpp:func:`nrf_cloud_connect` requires a list of supported features in the application.
-Use the following code in your application to initialize this list:
+The application can use :cpp:func:`nrf_cloud_connect` to connect to the cloud.
+This API triggers a series of events and actions in the system.
+If the API fails, the application must retry to connect.
 
-
-.. code-block:: c
-
-   /* Array of supported user association methods. */
-   const enum nrf_cloud_ua supported_uas[] = {
-		NRF_CLOUD_UA_BUTTON
-   };
-
-   /* Array of supported sensors. */
-   const enum nrf_cloud_sensor supported_sensors[] = {
-		NRF_CLOUD_SENSOR_GPS,
-		NRF_CLOUD_SENSOR_FLIP
-   };
-
-   /* List of supported user association methods. */
-   const struct nrf_cloud_ua_list ua_list = {
-		.size = ARRAY_SIZE(supported_uas),
-		.ptr = supported_uas
-   };
-
-   /* List of supported sensors. */
-   const struct nrf_cloud_sensor_list sensor_list = {
-		.size = ARRAY_SIZE(supported_sensors),
-		.ptr = supported_sensors
-   };
-
-   /* Struct containing both lists. */
-   const struct nrf_cloud_connect_param param = {
-		.ua = &ua_list,
-		.sensor = &sensor_list,
-   };
-
-   int err = nrf_cloud_connect(&param);
-
-
-First, the library tries to establish the transport for communicating with the cloud. This procedure involves a TLS handshake that might take up to three seconds. The API blocks for the duration of the handshake.
+First, the library tries to establish the transport for communicating with the cloud.
+This procedure involves a TLS handshake that might take up to three seconds.
+The API blocks for the duration of the handshake.
 
 Next, the API subscribes to an MQTT topic to start receiving user association requests from the cloud.
+The cloud uses the device's certificates for authentication.
```

Comment:
Should there be some information on how to replace the certificates on the board?
And/or a link to https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/include/modem/modem_key_mgmt.html or https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrfxlib/bsdlib/doc/security_tags.html ?

Reply:
this is the most useful/relevant link:
https://devzone.nordicsemi.com/nordic/cellular-iot-guides/b/getting-started-cellular/posts/nrf-cloud-certificate-update

Reply:
That is from the user perspective - but developers that include the nRF Cloud library probably want to automatize this and not do it manually for every device. Or what is the expected user workflow here?

Reply:
workflow would depend on the user and their stage of development.
please suggest how the desired link should be formatted/inserted.

Reply:
```suggestion
The cloud uses the device's certificates for authentication.
See `nRF Connect for Cloud Certificate Update`_ and the :ref:`modem_key_mgmt` library for more information on modem credentials.
```

And add the following line to nrf/doc/nrf/links.txt (in the "Links to the Nordic website" section):
```
.. _`nRF Connect for Cloud Certificate Update`: https://devzone.nordicsemi.com/nordic/cellular-iot-guides/b/getting-started-cellular/posts/nrf-cloud-certificate-update
```